src/main/java/com/iri/dao3/dao/jdbc/DriverDaoImpl.java

```diff
@@ -0,0 +1,116 @@
+package com.iri.dao3.dao.jdbc;
+
+import com.iri.dao3.dao.DriverDao;
+import com.iri.dao3.exception.DataProcessingException;
+import com.iri.dao3.lib.Dao;
+import com.iri.dao3.model.Driver;
+import com.iri.dao3.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String insertQuery = "INSERT INTO drivers (name, license_number) VALUES(?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement =
+                        connection.prepareStatement(insertQuery, Statement.RETURN_GENERATED_KEYS)
+        ) {
+            preparedStatement.setString(1, driver.getName());
+            preparedStatement.setString(2, driver.getLicenceNumber());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                driver.setId(resultSet.getObject(1, Long.class));
+            }
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get connection in method create Driver", e);
+        }
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String selectQuery = "SELECT * FROM drivers WHERE deleted = false AND id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(selectQuery)
+        ) {
+            preparedStatement.setLong(1, id);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            Driver driver = null;
+            while (resultSet.next()) {
+                return constructDriverFromResultSet(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't receive driver in method get", e);
```

Comment:
```suggestion
            throw new DataProcessingException("Could not get driver by id " + id, e);
```
You get the point, check you other methods))

Reply:
ok